vcluster/_partials/manage/deploy-changes.mdx

```diff
@@ -95,22 +95,36 @@ Replace:
 Learn more about [Cluster API Provider for vCluster](https://github.com/loft-sh/cluster-api-provider-vcluster).
 
 <Flow>
+<Step>
+
+   [Install](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) the `clusterctl` CLI.
+
+</Step>
+<Step>
+
+   Install the vCluster provider.
+
+   ```bash
+   clusterctl init --infrastructure vcluster:v0.2.0
+   ```
+</Step>
 <Step>
 
    Export environment variables to be used by the cluster API provider to create an updated manifest. The manifest will be applied to your Kubernetes
    cluster, which will update your vCluster with the updated configuration options.
    
    ```bash
     # Replace the vCluster variables with the values of the vCluster that you are updating
-    export CLUSTER_NAME=$VCLUSTER_NAME
-    export CLUSTER_NAMESPACE=$VCLUSTER_NAMESPACE
-    # Since the vcluster.yaml has changed, you'll need to export it to the updated vcluster.yaml
+    export CLUSTER_NAME=<cluster name>
```

Comment:
I would make it `<VCLUSTER_NAME>` as it's not a cluster, it's the vCluster. And I would capitalize all the letters.

Also, if you're going to updated it in this section, I would do it across all the deployment methods in this section to keep it consistent for the variable you need to replace.

Reply:
I guess we must find a compromise between vCluster-specific wording and the "contract" that CAPI enforces regarding naming: https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#variables-1
I think it's common practice to use the CLUSTER_NAME variable, as we actually refer to CAPI's notion of a [Cluster](https://github.com/loft-sh/cluster-api-provider-vcluster/blob/main/templates/cluster-template.yaml#L2), even though it only "wraps" a virtual one in our case. If you look at other providers they also use the CLUSTER_NAME:
- https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/templates/cluster-template.yaml#L4
- https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/templates/cluster-template.yaml#L7

I guess it boils down if we want to diverge from the common naming or stick with it, with the risk of being slightly confusing.

Reply:
No, I don't want to change the naming of the environment variable. That is not my request comment about. That is standard expectations in CAPI. 

My review was related to the changes that were in this PR where it changes from specifically `$VCLUSTER_NAME` as the input to the `CLUSTER_NAME` variable and calls it `<cluster name>`

It should be `<VCLUSTER_NAME>`.

Reply:
Ah, got it. Sorry, did interpret your suggestion the wrong way.